doc/ref_model/chapters/chapter07.md

```diff
@@ -339,6 +339,13 @@ A good place to start to understand the requirements is to use the widely accept
 Additional Cloud Infrastructure security principles that need to be incorporated:
 * Authenticity – The ability to confirm the users are in fact valid users with the correct rights to access the systems or data.
 
+In mobile network field, the GSM Association ([GSMA](https://www.gsma.com/)) and its Fraud and Security working group developed a strong expertise to secure the global mobile ecosystem. 
```

Comment:
Can we express it as that the group consists of experts rather than have developed expertise and that they have developed a set of valuable documents.
```suggestion
In mobile network field, the GSM Association ([GSMA](https://www.gsma.com/)) and its Fraud and Security working group of experts have developed a set of documents specifying how to secure the global mobile ecosystem. 
```